assembl/static2/package.json

```diff
@@ -82,6 +82,7 @@
     "immutable": "~3.7.4",
     "isomorphic-fetch": "^2.2.1",
     "json-loader": "^0.5.4",
+    "linkifyjs": "^2.1.5",
```

Comment:
Does this pull in all of the dependencies, including the react module, jquery module, html, and others?

Reply:
Hmm, no doesn't pull jQuery dependency in (thank god). http://soapbox.github.io/linkifyjs/docs/2.1.html

Reply:
But, possible to just install `linkify-html` standalone?

Reply:
I need `linkify.find` so I suppose not